Discovery/Discovery/Presentation/DiscoveryView.swift

```diff
@@ -25,8 +31,11 @@ public struct DiscoveryView: View {
         .accessibilityElement(children: .ignore)
         .accessibilityLabel(DiscoveryLocalization.Header.title1 + DiscoveryLocalization.Header.title2)
     
-    public init(viewModel: DiscoveryViewModel) {
+    public init(viewModel: DiscoveryViewModel, router: DiscoveryRouter, searchQuery: String? = nil, fromStartupScreen: Bool = false) {
         self._viewModel = StateObject(wrappedValue: { viewModel }())
+        self.router = router
+        self.fromStartupScreen = fromStartupScreen
+        viewModel.searchQuery = searchQuery
     }
```

Comment:
SwiftLint. Line Length Violation: Line should be 120 characters or less; currently it has 134 characters (line_length)
->
```
    public init(
        viewModel: DiscoveryViewModel,
        router: DiscoveryRouter,
        searchQuery: String? = nil,
        fromStartupScreen: Bool = false
    ) {
        self._viewModel = StateObject(wrappedValue: { viewModel }())
        self.router = router
        self.fromStartupScreen = fromStartupScreen
        viewModel.searchQuery = searchQuery
    }
```

Reply:
Done